### PR TITLE
DOC: some general doc/sphinx fixes

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

File: doc/source/computation.rst [hunk 1]
```diff
@@ -428,15 +428,13 @@ Using a non-regular, but still monotonic index, rolling with an integer window d
...
-   dft = DataFrame({'B': [0, 1, 2, np.nan, 4]},
-                   index = pd.Index([pd.Timestamp('20130101 09:00:00'),
-                                     pd.Timestamp('20130101 09:00:02'),
-                                     pd.Timestamp('20130101 09:00:03'),
-                                     pd.Timestamp('20130101 09:00:05'),
-                                     pd.Timestamp('20130101 09:00:06')],
-                                    name='foo'))
-
+   dft = pd.DataFrame({'B': [0, 1, 2, np.nan, 4]},
+                      index = pd.Index([pd.Timestamp('20130101 09:00:00'),
```
Comment (jreback) [inline doc/source/computation.rst]: do we still have ``*`` imports anywhere? (otherwise this *should* have broken)

**Comment by jreback** [comment]
lgtm

File: doc/source/computation.rst (hunk 1 above)
Comment (jorisvandenbossche) [inline doc/source/computation.rst]: It *was* broken :-) (as I noticed in the travis doc build log, otherwise I wouldn't have catched this)

Comment (jreback) [inline doc/source/computation.rst]: haha, makes sense! good that these raise (though didn't seem to fail the travis build) :<

Comment (jorisvandenbossche) [inline doc/source/computation.rst]: no, currently the travis build does not fail for such errors. But in principle we could do a search for certain patterns in the doc build log and let travis fail based on that

**Comment by jorisvandenbossche** [comment]
@jreback there were some lint issues introduced in https://github.com/pydata/pandas/pull/13710 (so travis for master will start to fail when it reaches that PR). But fixed them here

**Pull request merged**